docs.html

```diff
@@ -0,0 +1,552 @@
+            </pre>
+        </div>
+
+        <!--Navigation-->
+        <h2 id="navigation">Navigation</h2>
+        <h4>Examples</h4>
+        <div class="div-example">
+            <nav class="navigation">
+                <div class="title-section">
+                    <h4>Brand Name</h4>
+                </div>
+                <div class="links-section">
+                    <ul class="links-list">
+                        <li class="links-list-item"><a href="#">Link 1</a></li>
+                        <li class="links-list-item"><a href="#">Link 2</a></li>
+                        <li class="links-list-item"><a href="#">Link 3</a></li>
+                    </ul>
+                </div>
+            </nav>
+        </div>
+        <h4>Code Snippet</h4>
+        <div class="code-snippet">
+            <pre>
+                <code>
+        &ltnav class="navigation"&gt
+            &ltdiv class="title-section"&gt
+                &lth4&gtBrand Name&lt/h4&gt
+            &lt/div&gt
+            &ltdiv class="links-section"&gt
+                &ltul class="links-list"&gt
+                    &ltli class="links-list-item"&gt&lta href="#"&gtLink 1&lt/a&gt&lt/li&gt
+                    &ltli class="links-list-item"&gt&lta href="#"&gtLink 2&lt/a&gt&lt/li&gt
+                    &ltli class="links-list-item"&gt&lta href="#"&gtLink 3&lt/a&gt&lt/li&gt
+                &lt/ul&gt
+            &lt/div&gt
+        &lt/nav&gt
+                </code>
+            </pre>
+        </div>
+        <!--Ratings-->
+        <h2 id="ratings">Ratings</h2>
+        <h4>Example</h4>
+        <div class="div-example">
+
+            <span class="material-icons rating-stars">
+                grade
+            </span>
+
+            <span class="material-icons rating-stars">
+                grade
+            </span>
+            <span class="material-icons rating-stars">
+                grade
+            </span>
+            <span class="material-icons rating-stars">
+                grade
+            </span>
+            <span class="material-icons rating-stars">
+                grade
+            </span>
+        </div>
+        <h4>Code Snippet</h4>
+        <div class="code-snippet">
+            <pre>
+                <code>
+            &ltspan class="material-icons rating-stars"&gt
+                grade
+            &lt/span&gt
+
+            &ltspan class="material-icons rating-stars"&gt
+                grade
+            &lt/span&gt
+            &ltspan class="material-icons rating-stars"&gt
+                grade
+            &lt/span&gt
+            &ltspan class="material-icons rating-stars"&gt
+                grade
+            &lt/span&gt
+            &ltspan class="material-icons rating-stars"&gt
+                grade
+            &lt/span&gt
+                </code>
+            </pre>
+        </div>
+
+        <!--Text-->
+        <h2 id="text">Text</h2>
+        <h4>Example</h4>
+        <div class="div-example">
```

Comment:
Along with the sample text, we can also show the size of the font for that particular text.